Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.Tests/ErrorTesting.cs

```diff
@@ -4,131 +4,134 @@
+        [Test]
+        public void UploadFiles_TestThrowsWithWrongPort()
         {
-            Directory = _workDir,
-            FileName = "SFTPUploadTestFile1.txt",
-            Action = SourceAction.Error,
-            Operation = SourceOperation.Move,
-            DirectoryToMoveAfterTransfer = Path.Combine(_workDir, "moved")
-        };
-        Directory.CreateDirectory(source.DirectoryToMoveAfterTransfer);
-
-        var ex = Assert.Throws<Exception>(() => SFTP.UploadFiles(source, _destination, connection, _options, _info, new CancellationToken()));
-        Assert.That(ex.Message.StartsWith($"SFTP transfer failed: 1 Errors: Failure in CheckIfDestination"));
-        Assert.IsTrue(File.Exists(Path.Combine(_source.Directory, _source.FileName)));
-
-        Directory.Delete(source.DirectoryToMoveAfterTransfer, true);
-    }
+            var connection = Helpers.GetSftpConnection();
+            connection.Port = 51651;
 
-    [Test]
-    public void UploadFiles_TestSourceMoveWithFileAlreadyInMovedFolder()
-    {
-        Helpers.UploadSingleTestFile(_destination.Directory, Path.Combine(_workDir, _source.FileName));
+            var ex = Assert.Throws<Exception>(() => SFTP.UploadFiles(_source, _destination, connection, _options, _info, new CancellationToken()));
+            Assert.That(ex.Message.StartsWith("SFTP transfer failed: Unable to establish the socket:"));
+        }
 
-        var connection = Helpers.GetSftpConnection();
-        var source = new Source
+        [Test]
+        public void UploadFiles_TestThrowsWithWrongAddress()
         {
-            Directory = _workDir,
-            FileName = "SFTPUploadTestFile1.txt",
-            Action = SourceAction.Error,
-            Operation = SourceOperation.Move,
-            DirectoryToMoveAfterTransfer = Path.Combine(_workDir, "moved")
-        };
-        Directory.CreateDirectory(source.DirectoryToMoveAfterTransfer);
-        File.Copy(Path.Combine(source.Directory, source.FileName), Path.Combine(source.DirectoryToMoveAfterTransfer, source.FileName));
-
-        var ex = Assert.Throws<Exception>(() => SFTP.UploadFiles(source, _destination, connection, _options, _info, new CancellationToken()));
-        Assert.That(ex.Message.Contains($"Error: Failure in source operation:"));
-        Assert.IsTrue(File.Exists(Path.Combine(_source.Directory, _source.FileName)));
-
-        Directory.Delete(source.DirectoryToMoveAfterTransfer, true);
-    }
+            var connection = Helpers.GetSftpConnection();
+            connection.Address = "local";
 
-    [Test]
-    public void UploadFiles_TestErrorMessage()
-    {
-        var connection = Helpers.GetSftpConnection();
-        connection.Password = "cuinbeu8i9ch";
+            var ex = Assert.Throws<Exception>(() => SFTP.UploadFiles(_source, _destination, connection, _options, _info, new CancellationToken()));
+            Assert.That(ex.Message.StartsWith("SFTP transfer failed: Unable to establish the socket:"));
```

Comment:
This should be `Assert.IsTrue` instead of `Assert.That` to make tests consistent.